<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QDialog" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>110</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>110</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_mean">
         <property name="text">
          <string>&amp;Mean</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_median">
         <property name="text">
          <string>M&amp;edian</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Size:</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_size</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_size">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_size">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_laplacian">
     <property name="text">
      <string>&amp;Laplacian sharpening</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_preview">
       <property name="text">
        <string>&amp;Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Filter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Filter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_size</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_size</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
